src/main/java/seedu/address/model/person/TelegramHandle.java

```diff
@@ -5,17 +5,18 @@
 
 /**
  * Represents a Person's address in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ * Guarantees: immutable; is valid as declared in {@link #isValidTelegramHandle(String)}
  */
-public class Address {
+public class TelegramHandle {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Telegram Handle must "
+            + "use a-z, 0-9 and underscores. Minimum length is 5 characters";
 
     /*
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
```

Comment:
Update the comments to match your new regex, to maintain readability and maintainability.